### ISSUE TITLE
RuntimeError: unreachable

### ISSUE THREAD

**Issue by viceice** [editorconfig/editorconfig-core-js#111]
```
RuntimeError: unreachable
    at null.<anonymous> (wasm://wasm/00051056:1:55953)
    at null.<anonymous> (wasm://wasm/00051056:1:62171)
    at null.<anonymous> (wasm://wasm/00051056:1:70209)
    at null.<anonymous> (wasm://wasm/00051056:1:70338)
    at null.<anonymous> (wasm://wasm/00051056:1:69214)
    at null.<anonymous> (wasm://wasm/00051056:1:17561)
    at null.<anonymous> (wasm://wasm/00051056:1:47453)
    at null.<anonymous> (wasm://wasm/00051056:1:58421)
    at null.<anonymous> (wasm://wasm/00051056:1:54915)
    at module.exports.parse_to_uint32array (/usr/src/app/node_modules/@one-ini/wasm/one_ini.js:235:14)
    at parseBuffer (/usr/src/app/node_modules/editorconfig/lib/index.js:58:52)
    at processFileContents (/usr/src/app/node_modules/editorconfig/lib/index.js:234:25)
    at getConfig (/usr/src/app/node_modules/editorconfig/lib/index.js:272:12)
    at getAllConfigs (/usr/src/app/node_modules/editorconfig/lib/index.js:310:24)
    at parse (/usr/src/app/node_modules/editorconfig/lib/index.js:440:21)
```

not sure why this happen

- ref https://github.com/renovatebot/renovate/issues/18540

```ini
#
# .editorconfig
#
# http://editorconfig.org
#
# The EditorConfig project consists of a file format for defining coding styles
# and a collection of text editor plugins that enable editors to read the file
# format and adhere to defined styles.
#

# top-most EditorConfig file
root = true

# Unix-style newlines with a newline ending every file
[*]
end_of_line = lf
indent_style = space
indent_size = 4
insert_final_newline = true

# package.json specific rules
[package.json]
indent_size = 2

# .yml specific rules
[*.yml]
indent_size = 2
```

**Comment by viceice** [comment]
Added config file, which is causing the error

**Comment by viceice** [comment]
It seems this file also triggers the issue (empty comment)

```ini
#
```

**Comment by hildjj** [comment]
Will look into this ASAP.

**Comment by hildjj** [comment]
Published in editorconfig@1.0.1

**Comment by viceice** [comment]
thanks for quick fix
